applications/2025/pending/sudiptasarkar.yml

```
@@ -0,0 +1,108 @@
+    description: Delivered commit-to-question semantic search functionality using LangChain and GitPython for GitRAG, enabling natural language understanding of version history.
+    time_taken: 3–4 days
+    users_impacted: 10–15 developer testers during internal rollout
+
+  weekly_commit_average: 15–20
+
+  friday_availability:
+    answer: try-and-stop-me
+    timezone: Asia/Kolkata
+
+  startup_experience:
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)